elements/pfe-button/src/pfe-button.scss

```
@@ -0,0 +1,138 @@
+// Please see the pfe-sass README for guidance on using these tools
+@import "../../pfe-sass/pfe-sass";
+
+$LOCAL: button;
+
+:host {
+  // Example local variable, using BEM format
+  // --pfe-button--Property: value;
+  display: inline-block;
+}
+
+:host([hidden]) {
+  display: none;
+}
+
+:host([pfe-variant="primary"]) button,
+button {
+  background-color: #06c;
+  color: #fff;
+  font-size: 16px;
+  padding: 8px 16px;
+  cursor: pointer;
+  border-radius: 3px;
```

You can find these values in `elements/pfe-sass/maps/_general.scss`:

```suggestion
  border-radius: #{pfe-var(surface--border-radius)};
```